packages/map-template/src/i18n/nl.js

```diff
@@ -0,0 +1,74 @@
+const nl = {
+    // Common
+    'Close': 'Sluit',
+    'From': 'Van',
+    'To': 'Naar',
+    'Distance': 'Afstand',
+    'Estimated time': 'Geschatte tijd',
+    'Level': 'Niveau',
+    'Search by name, category, building...': 'Zoeken op naam, categorie, gebouw…',
+    'Nothing was found': 'Er is niets gevonden',
+    'd': 'd',
+    'h': 'u',
+    'min': 'min',
+    'Walk': 'Wandelen',
+    'Bike': 'Fietsen',
+    'Transit': 'Overgang',
+    'Drive': 'Rijden',
+    'Start wayfinding': 'Begin met navigeren ',
+    // Location Details
+    'Read full description': 'Lees de volledige beschrijving',
+    // Locations List
+    'Locations': 'Locaties',
+    'Directions not available': 'Routebeschrijving niet beschikbaar',
+    'Start directions': 'Start routebeschrijving',
+    // Route instructions
+    'Previous': 'Vorig',
+    'Next': 'Volgende',
+    'StepYofX': 'Stap {{activeStep}} van {{totalSteps}}',
+    // Route Instructions Step
+    'You have arrived': 'Je bent aangekomen',
+    'Leave': 'Verlaat',
+    'Park': 'Parkeer',
+    'at': 'bjj',
```

Comment:
My secret informer points this as a typo. Could you please check if bjj is right word for it?
```suggestion
    'at': 'bij',
```